internal/grpc/server.go

```diff
@@ -44,7 +44,7 @@ func (g *Server) GetBucket(ctx context.Context, req *pb.GetBucketRequest) (*pb.B
 		TimeCreated:           timestamppb.New(bucket.TimeCreated),
 	}
 	return grpc_bucket, nil
-	///return GetBucketFromBackend(g.backend, req.Bucket)
+	// /return GetBucketFromBackend(g.backend, req.Bucket)
```

Comment:
I realize that this was autoformating, but do you want to take the opportunity to delete this line altogether 😁

Reply:
Sure thing!